integrations/weights_bias/src/haystack_integrations/components/connectors/weave_connector.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Dict, Optional
+
+from haystack import component, default_from_dict, default_to_dict, tracing
+from haystack.tracing import tracer as haystack_tracer
+
+from haystack_integrations.tracing.weave import WeaveTracer
+
+
+@component
+class WeaveConnector:
+    """
+    Collects traces from your pipeline and sends them to Weights & Biases.
+
+    Add this component to your pipeline to integrate with the Weights & Biases Weave framework for tracing and
+    monitoring your pipeline components.
+    """
+
+    def __init__(self, pipeline_name: str) -> None:
+        """
+        Initialize WeaveConnector.
+
+        :param pipeline_name: The name of the pipeline you want to trace.
+        """
+        self.pipeline_name = pipeline_name
+
+        # TODO: This is a hack because content tracing enabled seems to rely on import order which is hard to debug.
+        # As a workaround, we assume that users adding the WeaveConnector to a pipeline always want to trace content.
+        haystack_tracer.is_content_tracing_enabled = True
```

Comment:
we should not do this: if users want to do content tracing, they should explicitly opt in 

[An example of how this is handled in Langfuse Tracer](https://github.com/deepset-ai/haystack-core-integrations/blob/65c46c09fbb26a9a1fb19776042fa16ed2dc5c61/integrations/langfuse/src/haystack_integrations/tracing/langfuse/tracer.py#L311)

Reply:
This was a bit confusing to me - I will adapt the langfuse tracer strategy